public/csp.js

```diff
@@ -150,6 +148,10 @@ mongoose.connect(dbConfig.db, {
 
 app.get("/", function (req, res, next) {
     return res.redirect('./home.html', { root: __dirname });
+
+    if (req.ok) {
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- a602d6517d4b2d5a9df2b4ae5292a980 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Unreachable, Unused or Dead Code
_Medium CWE-561_
Code block is unreachable because of the syntactic structure of the code

Actions intended to be performed by the unreachable code will never occur.

### How to fix
Unreachable and dead code should be analyzed and either removed or corrected depending on whether it is needed. If the code should execute but doesn't, any bugs preventing the code from executing must be fixed. If the code is not needed, and its removal will not have a negative impact on the application, it should be removed.